src/box/CMakeLists.txt

```diff
@@ -77,6 +77,9 @@ set_property(DIRECTORY PROPERTY ADDITIONAL_MAKE_CLEAN_FILES ${lua_sources})
 include_directories(${ZSTD_INCLUDE_DIRS})
 include_directories(${PROJECT_BINARY_DIR}/src/box/sql)
 include_directories(${PROJECT_BINARY_DIR}/src/box)
+include_directories(${EXTRA_BOX_INCLUDE_DIRS})
+# Explicitly include current directory for out-of-tree source files.
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Can you leave an example here? I don't spot what is 'current directory' in this context.

Reply:
It's `src/box`. Needed so that EE sources can include files located in `src/box` with `#include "foo.h"`, as if they themselves were located in `src/box`.

Reply:
IMO that should be moved into `EXTRA_BOX_INCLUDE_DIRS`.

Reply:
@alyapunov suggested to move to `EXTRA_BOX_INCLUDE_DIRS` defined in the EE CMake config, which I did. So this comment isn't relevant anymore.